src/Handler/AddPrefixCommandHandler.php

```diff
@@ -22,6 +26,30 @@
 class AddPrefixCommandHandler
 {
     /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    /**
+     * @var Finder
+     */
+    private $finder;
+
+    /**
+     * @var Scoper
+     */
+    private $scoper;
+
+    public function __construct(Filesystem $filesystem, Finder $finder)
```

Comment:
Yes, I thought it would be easier to test the CommandHandler by mocking the dependencies, but the tests are using the real Filesystem and Finder, so we can construct them internally